doc/source/conf.py

```diff
@@ -86,9 +86,9 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'GA4GH Schemas'
-copyright = u'2015, Global Alliance for Genomics and Health'
-author = u'Global Alliance for Genomics and Health'
+project = 'CanDIG Schemas'
+copyright = '2015, Global Alliance for Genomics and Health'
+author = 'Global Alliance for Genomics and Health'
```

Comment:
this should be changed as well:
GA4GH + CanDIG maybe

Reply:
the author or copyright?